tumblrWatch/com/5yp/tumblrWatch/5yp_test_post.py

```diff
@@ -0,0 +1,25 @@
+import pytumblr
```

Comment:
This is not a program containing tests so please rename it to describe what it actually does.